### PR TITLE
[#61] Removed tasty-golden

### PR DESCRIPTION
**Pull request by mgttlinger**

### CODE REVIEW

File: src/Importify/Main/File.hs [hunk 1]
```diff
@@ -78,12 +80,24 @@ importifyFileOptions options srcFile = do
...
+  toFilePath :: Path b t -> FilePath
+  toAbs :: Path Abs Dir -> Path b t -> Path Abs t
+
+instance PathHandling Rel File where
+  toFilePath = fromRelFile
+  toAbs = (</>)
+
+instance PathHandling Abs File where
+  toFilePath = fromAbsFile
+  toAbs _ p = p
```
Comment (ChShersh) [inline src/Importify/Main/File.hs]: This looks like a dirty hack to me. `toFilePath` is already defined in `Path` module for `Path b t`. I think it's better to pass always `Path Abs File` to `importifyFileContent` and do `getCurrentDir` outside `importifyFileContent`

File: test/GGenerator.hs [hunk 2]
```diff
@@ -6,13 +6,14 @@
 
 module Main where
 
-import           Universum
+import           Universum  hiding (writeFile)
```
Comment (ChShersh) [inline test/GGenerator.hs]: `writeFile` from `universum` is just:

```haskell
writeFile :: MonadIO m => FilePath -> Text -> m ()
writeFile a b = liftIO (Data.Text.IO.writeFile a b)
```

So I don't see reasons to hide this `writeFile` and use it directly from `Data.Text.IO`.

**Comment by ChShersh** [comment]
@mgttlinger Thanks for your contribution! I requested only couple changes in your code. But in general it looks really great! :slightly_smiling_face:

File: test/GGenerator.hs (hunk 2 above)
Comment (mgttlinger) [reply test/GGenerator.hs]: I should have looked at the `writeFile` there. I just used the `Text.IO` one and fixed the resulting error.

**Comment by mgttlinger** [comment]
Btw what does `t&b` stand for?

**Comment by ChShersh** [comment]
@mgttlinger `t&b` is for `tests & benchmarks` :) All your changes look good to me! Thanks for your great contribution!

**Pull request merged**